src/lib/filtered_external_transition/filtered_external_transition.ml

```diff
@@ -92,11 +92,18 @@ let participant_pks
 
 let commands {transactions= {Transactions.commands; _}; _} = commands
 
-let validate_transactions external_transition =
+let validate_transactions ((transition_with_hash, _validity) as transition) =
   let staged_ledger_diff =
-    External_transition.Validated.staged_ledger_diff external_transition
+    External_transition.Validated.staged_ledger_diff transition
+  in
+  let external_transition = With_hash.data transition_with_hash in
+  let coinbase_receiver =
+    External_transition.protocol_state external_transition
+    |> Coda_state.Protocol_state.consensus_state
+    |> Consensus.Data.Consensus_state.coinbase_receiver
```

Comment:
Can this be replaced with `External_transition.coinbase_receiver external_transition`?

Reply:
thanks, I missed this one